src/eos_gates.js

```
@@ -183,6 +183,27 @@ function installAppFromStore(remote, appId) {
     spawnProcess(['gnome-software', '--details=%s'.format(appStoreId)]);
 }
 
+function launchFlatpakApp(appId) {
+    try {
+        spawnProcess(['flatpak', 'run', appId]);
```

I've tried that and found that it just silently failed.